frontend/packages/console-app/src/components/guided-tours/GuidedTourItem.tsx

```diff
@@ -0,0 +1,38 @@
+import * as React from 'react';
+import { CatalogTile } from '@patternfly/react-catalog-view-extension';
+import { GuidedTourItem, TourStatus } from './utils/guided-tour-typings';
+import TourItemHeader from './TourItemHeader';
+import TourItemDescription from './TourItemDescription';
+import TourItemFooter from './TourItemFooter';
+import './GuidedTourItem.scss';
+
+type GuidedTourItemProps = GuidedTourItem & TourStatus;
+
+const GuidedTourItem: React.FC<GuidedTourItemProps> = ({
+  iconURL,
+  altIcon,
+  name,
+  description,
+  status,
+  active,
+  duration,
+  prerequisites,
+}) => (
+  <CatalogTile
+    className="odc-guided-tour-item"
+    featured={active}
+    title={
+      <TourItemHeader
+        iconURL={iconURL}
+        altIcon={altIcon}
+        name={name}
+        status={status}
+        duration={duration}
```

Comment:
Shouldn't this should part of the description?

Reply:
the description doesn't support those badges without CSS overrriding.
![Screenshot from 2020-06-22 16-40-12](https://user-images.githubusercontent.com/24852534/85288866-1a810f00-b4b4-11ea-9ef0-e84a0071cf49.png)

Reply:
My point is why is status and duration in the header and not the description?

Reply:
The description prop of the catalog tile is not supporting the badges